Hack4Impact-Princeton/alp-portal
# 290 m edit profile

created edit profile modal with a few modifications compared to the figma

changes:
- the edit profile button couldn't go next to the title of the page 
- I made the profile picture upload button the same as it is on the normal page and changed the color to fit the scheme
- there is not yet support for hobbies, affiliation, and fav book because it doesn't exist in the database yet

to test:
1. log into test3
2. go to the profile page
3. click edit profile (inside of the first card)
4. modify the information, but don't click submit yet - click the "x" in the top right corner or click cancel. The newly input information shouldn't be saved, and should neither appear on the profile page nor appear on the profile page after reloading the page
5. modify the information and click submit - the changes should be reflected instantly.

let me know if I should add support for hobbies, affiliation, and fav book through modifications to the schema
